lib/connection/statement.js

```diff
@@ -1113,10 +1117,17 @@ function sendRequestPreExec(statementContext, onResultAvailable)
 
   // build the basic json for the request
   var json =
-    {
-      disableOfflineChunks: false,
-      sqlText: statementContext.sqlText
-    };
+  {
+    disableOfflineChunks: false,
+  };
+  if (!statementContext.resubmitRequest)
+  {
+    json.sqlText = statementContext.sqlText;
+  }
+  else
+  {
+    json.sqlText = 'select 1;';
```

Comment:
why we are hard coding SQL here?

Reply:
If the user only passes a requestId when executing a query, the request fails due to not having a sqlText. The hardcoded SQL is for when the user only passes a requestId.

I've been discussing with Kris if it's possible to not have a sqlText when a requestId is already provided so that adding SQL wouldn't be necessary